.changeset/many-bags-press.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': minor
```

Comment:
I think this is probably safe to release as a patch. It fixes incorrect sorting in some edge cases, so probably needn’t be tagged as a minor.
```suggestion
'@astrojs/starlight': patch
```